hack/verify-toc.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# keep in sync with hack/update-toc.sh
```

Comment:
I think you're missing that file?

Reply:
ya this is just a quick test of this script borrowed from k8s repo.. if it works.. we can add this in the validation job and then not worry about ensuring TOCs are updated while reviewing doc PRs if everyone agrees this is a good thing to have .. will update properly and remove the wip

Reply:
oh just saw the license header...

btw, I would suggest moving the version number / sha to a separate file (e.g. mdtoc-version) and then reading it from both shell scripts with `TOOL_VERSION=$(head mdtoc-version)`. This way it it always in-sync.

otherwise idea LGTM. I am just surprised that they chose to go with a `.notableofcontents` file instead of the opposite (`.tableofcontents`) because now we have to add new files explicitly to the list if we don't want a generated TOC. I also wish there was a better way of installing these temporary Go binaries without updating go.mod / go.sum.